### PR TITLE
burn: remove padding to match layout figma

### LINKED ISSUE #141: Bug layout app not match with figma
App: 
image

Figma: 
image

The buttons have to stay in space between without padding

### PR DESCRIPTION
**Pull request by rodrigoodev**
Closes #141 

## Description
Matching to layout figma

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by SauveJeanLuc** [comment]
@rodrigoodev  Can you use both screenshots to help spot changes you've made ?

**Comment by SauveJeanLuc** [comment]
@rodrigoodev  I think it's better the way it was, and the Figma has to be corrected instead.

WDYT ?

**Comment by rodrigoodev** [comment]
hello @SauveJeanLuc here a print of master
image

the padding not matching with figma in master, i correct this padding 
the non-aligned padding was annoying me :smile:

**Comment by SauveJeanLuc** [comment]
> nielsen's law

Interesting, could you tell us more about the Nielsen's law ?
It might help us take the decision.

**Comment by rodrigoodev** [comment]
> 
Here is a link about [heuristics of NIelsen](https://www.nngroup.com/articles/ten-usability-heuristics/),
In this case, I'm talking about the alignment between the container divs, when padding is applied to the div, misalignment happens, which causes discomfort in design.

If it climbed, it could turn into an accordion effect.

for example the image below
image

if you don't have a design knowledge, it's ok to keep the page structure like this, and it's quite normal when you're starting and you don't have a sense of standardization that you can't see these details in smaller design, as is the case with our whole list.


Now look at the difference when we apply Nielsen's law 4, law of organization and standardization.
image

You feel the lighter design, right?

now, putting the designs next to each other, you will get a better feeling of looking at the design on the right.
image

So basically that's why my PR.

Now feel free to decide which one you think is best.

**Comment by SauveJeanLuc** [comment]
Great, Idea accepted.
Pull from master and I'll merge

**Review by SauveJeanLuc — APPROVED** [review]

**Comment by SauveJeanLuc** [comment]
Check #62
